bridge_adaptivity/bridge_lti/urls.py

```diff
@@ -1,10 +1,19 @@
+"""
+Launch and LTI urls.
```

Comment:
why do you mention launch separately from the LTI?

Reply:
I fixed this.